### PR TITLE
Update resource-management.md

### PR DESCRIPTION
**Pull request by xboxeer**
## Summary

The check resource existence part is outdated, updated with latest method

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by xboxeer** [comment]
@fengzhou-msft kindly help to review,

File: docs/azure/sdk/resource-management.md [hunk 1]
```diff
@@ -237,9 +236,9 @@ Console.WriteLine(availabilitySet.Data.Name);
 
 ### Check if a [Resource] exists
 
-If you aren't sure if a resource you want to get exists, or you just want to check if it exists, you can use `GetIfExists()` or `CheckIfExists()` methods, which can be invoked from any [Resource]Collection class.
+If you aren't sure if a resource you want to get exists, or you just want to check if it exists, you can use `Exists()` or `ExistsAsync()` methods, which can be invoked from any [Resource]Collection class.
 
-`GetIfExists()` and `GetIfExistsAsync()` return a `Response<T>`, where `T` is null if the specified resource does not exist. On the other hand, `CheckIfExists()` and `CheckIfExistsAsync()` return `Response<bool>` where the Boolean value is false if the specified resource does not exist. Both of these methods still give you access to the underlying raw response.
+`Exists()` returns a `Response<bool>` while `ExistsAsync()` as its async version returns a `Task<Response<bool>>`, in the `Reponse<bool>` object, you can visit its `Value` property to check if a [Resource] exists. The `Value` is `false` if the resource does not exist and vice versa.
```
Comment (fengzhou-msft) [inline docs/azure/sdk/resource-management.md:241]:
```suggestion
`Exists()` returns a `Response<bool>` while `ExistsAsync()` as its async version returns a `Task<Response<bool>>`. In the `Reponse<bool>` object, you can visit its `Value` property to check if a [Resource] exists. The `Value` is `false` if the resource does not exist and vice versa. 
```

File: docs/azure/sdk/resource-management.md [hunk 2]
```diff
@@ -266,7 +265,7 @@ ArmClient armClient = new ArmClient(new DefaultAzureCredential());
 Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
 string rgName = "myRgName";
 
-bool exists = await subscription.GetResourceGroups().CheckIfExistsAsync(rgName);
+bool exists = await subscription.GetResourceGroups().Exists(rgName).Value;
```
Comment (fengzhou-msft) [inline docs/azure/sdk/resource-management.md:268]: `.Value` is not necessary as `Response<bool>` can be converted to `bool` implicitly here.

**Review by CamSoper — APPROVED** [review]
~LGTM! Thanks, @fengzhou-msft!~

Correction: I misunderstood what I was looking at. It's Friday.

File: docs/azure/sdk/resource-management.md (hunk 2 above)
Comment (CamSoper) [reply docs/azure/sdk/resource-management.md:268]: @fengzhou-msft Other than that, you feel good about @xboxeer's changes? If so, I'll just make this one and merge.

**Review by CamSoper — CHANGES_REQUESTED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Comment by xboxeer** [comment]
@CamSoper kindly help merge this PR, thanks

**Review by CamSoper — APPROVED** [review]

**Pull request merged**